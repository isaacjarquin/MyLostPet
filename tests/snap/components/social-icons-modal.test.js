import "react-native"
import React from "react"
import SocialIconsModal from "../../../components/social-icons-modal"

import renderer from "react-test-renderer"

it("renders correctly", () => {
	const tree = renderer.create(
		<SocialIconsModal />
	).toJSON()
	expect(tree).toMatchSnapshot()
})
