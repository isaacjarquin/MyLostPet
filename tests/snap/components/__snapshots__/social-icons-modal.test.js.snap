// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "justifyContent": "flex-start",
    }
  }
>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#333333",
        "borderTopLeftRadius": 3,
        "borderTopRightRadius": 3,
        "marginLeft": 15,
        "marginRight": 15,
        "marginTop": 5,
        "opacity": 0.8,
        "padding": 8,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "alignSelf": "center",
          "color": "white",
          "fontSize": 18,
        }
      }
    >
      Share
    </Text>
  </View>
  <Modal
    animationType="none"
    hardwareAccelerated={false}
    onModalHide={[Function]}
    onRequestClose={[Function]}
    transparent={true}
    visible={false}
  >
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hitSlop={undefined}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "black",
          "bottom": 0,
          "height": 1334,
          "left": 0,
          "opacity": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "width": 750,
        }
      }
      testID={undefined}
    />
    <View
      collapsable={undefined}
      onModalHide={[Function]}
      pointerEvents="box-none"
      style={
        Object {
          "backgroundColor": "#333333",
          "borderTopLeftRadius": 3,
          "borderTopRightRadius": 3,
          "flex": 1,
          "flexDirection": "column",
          "justifyContent": "center",
          "margin": 37.5,
          "marginTop": 500,
          "transform": Array [
            Object {
              "translateY": 0,
            },
          ],
        }
      }
    >
      <View
        style={
          Object {
            "backgroundColor": "#333333",
            "borderTopLeftRadius": 3,
            "borderTopRightRadius": 3,
            "flexDirection": "row",
            "marginLeft": 8,
            "marginTop": 10,
          }
        }
      >
        <Image
          source={1}
          style={
            Object {
              "height": 55,
              "marginLeft": 8,
              "width": 55,
            }
          }
        />
        <Image
          source={1}
          style={
            Object {
              "height": 55,
              "marginLeft": 8,
              "width": 55,
            }
          }
        />
        <Image
          source={1}
          style={
            Object {
              "height": 55,
              "marginLeft": 8,
              "width": 55,
            }
          }
        />
        <Image
          source={1}
          style={
            Object {
              "height": 55,
              "marginLeft": 8,
              "width": 55,
            }
          }
        />
        <Image
          source={1}
          style={
            Object {
              "height": 55,
              "marginLeft": 8,
              "width": 55,
            }
          }
        />
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        collapsable={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        nativeID={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "marginTop": 10,
            "opacity": 1,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          disabled={false}
          ellipsizeMode="tail"
          style={
            Object {
              "alignSelf": "center",
              "color": "white",
              "fontSize": 18,
            }
          }
        >
          Cerrar
        </Text>
      </View>
    </View>
  </Modal>
</View>
`;
